.drone.yml

```diff
@@ -288,7 +288,7 @@ name: translations
 
 platform:
   os: linux
-  arch: amd64
+  arch: arm64
```

Comment:
Why must change arch?

Reply:
The arm server has more core (and more memory) even if they are slower individually currently jobs are waiting for a slot on the amd64 server.